test/Rentals.spec.ts

```diff
@@ -2187,7 +2324,25 @@ describe('Rentals', () => {
       await evmIncreaseTime(daysToSeconds(offerParams.rentalDays))
       await evmMine()
 
-      await expect(rentals.connect(tenant).claim(land.address, tokenId)).to.be.revertedWith('Rentals#claim: NOT_LESSOR')
+      await expect(rentals.connect(tenant).claim([land.address], [tokenId])).to.be.revertedWith('Rentals#claim: NOT_LESSOR')
+    })
+
+    it('should revert when the contract address array and token ids array length dont match', async () => {
```

Comment:
I would add a new test like "Should revert when trying to claim a valid + invalid asset or not owned asset".

Reply:
Added tests